gitCoreJGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreRepository.java

```diff
@@ -85,21 +86,33 @@ public GitCoreRemoteBranch getRemoteBranch(String branchName) throws GitCoreExce
 
   @Override
   public List<IGitCoreLocalBranch> getLocalBranches() throws GitCoreException {
+    LOG.debug(() -> "Enter getLocalBranches for repository ${mainDirectoryPath} (${gitDirectoryPath})");
+    LOG.debug("List of local branches branches:");
     return Try.of(() -> getJgitGit().branchList().call())
         .getOrElseThrow(e -> new GitCoreException("Error while getting list of local branches", e))
         .stream()
         .filter(branch -> !branch.getName().equals(Constants.HEAD))
+        .map(branch -> {
+          LOG.debug(() -> "* ${branch.getName()}");
+          return branch;
+        })
         .map(ref -> new GitCoreLocalBranch(/* repo */ this,
             ref.getName().replace(GitCoreLocalBranch.BRANCHES_PATH, /* replacement */ "")))
         .collect(List.collector());
   }
 
   @Override
   public List<IGitCoreRemoteBranch> getRemoteBranches() throws GitCoreException {
+    LOG.debug(() -> "Enter getRemoteBranches for repository ${mainDirectoryPath} (${gitDirectoryPath})");
+    LOG.debug("List of remote branches branches:");
     return Try.of(() -> getJgitGit().branchList().setListMode(ListBranchCommand.ListMode.REMOTE).call())
         .getOrElseThrow(e -> new GitCoreException("Error while getting list of remote branches", e))
         .stream()
         .filter(branch -> !branch.getName().equals(Constants.HEAD))
+        .map(branch -> {
+          LOG.debug(() -> "* ${branch.getName()}");
```

Comment:
This asterisk is to make branches look like they were on a pointed list, right?

Reply:
yes